### ISSUE TITLE
Cupra Born

### ISSUE THREAD

**Issue by initdebugs** [tillsteinbach/VWsFriend#218]
Hi, it's been some time since we tried connecting to the Cupra API. There has been an update to the My Cupra app. Would you like to try again? Would it be a lot of work to adjust the VWsFriend project to also support the Cupra API? Thanks

**Comment by tillsteinbach** [comment]
Is the new version already ready for the MEB 3.0 software release?

**Comment by initdebugs** [comment]
That's okay! Just let me know :)

**Comment by initdebugs** [comment]
Hey, want to check again?

**Comment by tillsteinbach** [comment]
I have had a look again. It is going "in the right direction" I can see that the correct endpoints are there, but most give a 500 error back.

**Comment by initdebugs** [comment]
Is that something you can fix or is that the Cupra API not working right?

**Comment by tillsteinbach** [comment]
As some endpoints are working and some not I would say it is something on their side. I really hope for a bigger makeover of the App when there is a significant number of cars with SW 3.0 in the field.

**Comment by tillsteinbach** [comment]
Anyway I already added the login mechanism for MyCupra to the core of VWsFriend. So this part is then already ready to go.

**Comment by ppiontek** [comment]
Thanks for this cool project! Did you have a recent look at the API for Cupra Born? I could probably support development with access to a Cupra Born with current software 3.2.

**Comment by bbr111** [comment]
Hi, any news here?

**Comment by tillsteinbach** [comment]
No, I have written VWsFriend when I had a VW. Getting a skoda was my motivation to start with CarConnectivity to allow for more brands.
There will be something like VWsFriend for CarConnectivity in the near future, but I need some more time to rewrite everything.
